module/zfs/zfs_ctldir.c

```diff
@@ -1049,11 +1044,6 @@ zfsctl_snapshot_unmount(char *snapname, int flags)
 }
 
 #define	MOUNT_BUSY 0x80		/* Mount failed due to EBUSY (from mntent.h) */
-#define	SET_MOUNT_CMD \
-	"exec 0</dev/null " \
-	"     1>/dev/null " \
-	"     2>/dev/null; " \
-	"mount -t zfs -n '%s' '%s'"
```

Comment:
It looks like the old `SET_MOUNT_CMD` would fail and potentially execute arbitrary code if the mountpoint were to contain a single quote.